examples/go-pipeline/go-pipeline.cue

```diff
@@ -0,0 +1,176 @@
+package ssf
+
+_REPOSITORY: *"ttl.sh" | string @tag(repository)
+_APP_IMAGE: *"\(_REPOSITORY)/go-build-test-ssf" | string @tag(appImage)
+
+pipeline: "pipeline-go-test": {
+	apiVersion: "tekton.dev/v1beta1"
+	kind:       "Pipeline"
+	metadata: name: "pipeline-go-test"
```

Comment:
These should be inherited from the top level. Might be able to use `cue trim ./examples/go-pipeline -s` to simplify things.

Reply:
Always miss this step. :)